frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.styled.tsx

```diff
@@ -20,3 +20,13 @@ export const Button = styled(BaseButton)<Props>`
       alpha(primaryColor, 0.8)};
   }
 `;
+
+export const FilterPopoverSeparator = styled.hr`
+  border: 0;
+  height: 0;
+  border-top: 1px solid ${color("border")};
+`;
+
+export const EmptyFilterPickerPlaceholder = styled.div`
```

Comment:
Mimics the `.PopoverBody--marginBottom` class that the filter pickers have. I didn't want to use the class name on the `div` to keep with our current standards